custom-metrics-stackdriver-adapter/deploy/production/adapter.yaml

```diff
@@ -1,3 +1,7 @@
+# A staging deployment of Custom Metrics - Stackdriver Adapter.
+# This can be using for testing the adapter or components that depend on it.
+# It must not be used for production purposes, instead the production
+# deployment, which uses the officially released image, should be used.
```

Comment:
This file is the production deployment (at least based on path), so this comment should be removed.

Reply:
Done